docs/markdown/Reference-manual.md

```diff
@@ -436,33 +436,46 @@ system) with the given name with `pkg-config` and [with
 CMake](Dependencies.md#cmake) if `pkg-config` fails. Additionally,
 frameworks (OSX only) and [library-specific fallback detection
 logic](Dependencies.md#dependencies-with-custom-lookup-functionality)
-are also supported. This function supports the following keyword
-arguments:
+are also supported.
+
+Dependencies can also be resolved by a fallback subproject.
+Versions of Meson before 0.55.0 will never automatically search for
+a fallback.  *Since 0.55.0* Meson can look for a wrap file or a directory
```

Comment:
IMHO it's not clear what 'automatically search' means here.

Maybe instead of repeating the mechanism here, it could just link to the doc we already have:
```
Dependencies can also be resolved by a fallback subproject. See
[Wrap documentation](Wrap-dependency-system-manual.md#provide-section) and `fallback` keyword argument.
```

Reply:
btw, that wrap documentation page explains why optional dependencies does not use automatic fallback. Would make sense to add a note there that it can now be forced by setting `fallback: true`.

Reply:
Actually the wrap doc mention the trick of setting `fallback: 'foo'` which is now equivalent of setting `fallback: true`.

Reply:
On one hand wrap files and subprojects are somewhat related, on the other hand I think the reference manual should be self contained. I adjusted the documentation according to your other suggestions though.